schemas/2018-08-01/Microsoft.Network.json

```diff
@@ -4,6 +4,163 @@
   "title": "Microsoft.Network",
   "description": "Microsoft Network Resource Types",
   "resourceDefinitions": {
+    "frontDoors": {
```

Comment:
>frontDoors [](start = 5, length = 10)

please add tests for the new resource types in tests/2018-08-01.

Reply:
I added some minimal tests.